mobile/src/index.ts

```diff
@@ -1,10 +1,41 @@
 import express, { Application, Request, Response } from 'express';
-import authRouter from './router/auth-router';
+import * as authAppController from './controller/auth-app-controller';
+import * as apn from 'apn';
 
 const app: Application = express();
 
 app.use(express.json());
 
-app.use('/auth-app', authRouter);
+app.use('/auth-app', authAppController.authAppSetupController);
+
+app.get('/notification',  (req: Request, res: Response) => {
+    var token = 'AAAAPx8g3qg:APA91bGPNh5eH8Ve_GuUBen_tkcrQnrAfVxUpl-09VLPKEJ5mrfbQ9Ql1S9qHnDszkqjZ0rylZIvjX4S9AR1gtw7rcuR-082q1-HEFxeSPK2uQWX4nWhvYYk1-BdrDH4WvYYLUxt-pvl';
```

Comment:
Is this ok to use plain text about token and key below etc? I will move all of those stuff into GHA secret in the following PR